lib/assessment.js

```diff
@@ -5,7 +5,18 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
+  // get the arguments, assign to args
+  var args = arguments;
 
+  var result = 0;
+
+  // iterate over args length and compare value to result
+  for (var i = 0; i < args.length; i++) {
+    if (result < args[i]) {
+      result = args[i];
+    }
+  }
+  return result;
```

Comment:
Ah, OK. I thought we received explicit instruction to not use 'arguments' without assigning that to a variable.